src/main/resources/wa-task-cancellation-ia-asylum.dmn

```diff
@@ -36,6 +36,176 @@
           <text></text>
         </outputEntry>
       </rule>
+      <rule id="DecisionRule_0xhe2ed">
+        <inputEntry id="UnaryTests_1smac6l">
+          <text>"awaitingReasonsForAppeal"</text>
```

Comment:
do we need to update the ticket to reflect what we actually have in the cancellation DMN mapping?